### DISCUSSION TITLE
Pull request in Github

### DISCUSSION THREAD

**Discussion by abarnsathrian** [The-Coding-Foxes/the-algorithms#7] [Q&A] [answered]
How do you create a pull request on GitHub?

**Comment by Sabarinathan07** [comment, marked as answer]
To create a pull request on GitHub, fork the repository, clone it locally, create a new branch, and make changes. Commit and push your changes to your fork. On GitHub, click "Compare & pull request" in the original repo, write a description, and submit the pull request for review.
